libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-beta-message/eo-beta-message.component.scss

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.watt-card {
+  position: relative;
+  border-radius: 4px;
+  display: block;
+  padding: calc(1.5 * var(--watt-space-m));
+  background: var(--bg-card);
+  box-shadow: 0px 1px 6px rgba(11, 60, 93, 0.12),
+    0px 4px 18px 3px rgba(46, 50, 52, 0.08);
+
+  &.watt-solid {
+    border: 1px solid var(--watt-color-neutral-grey-300);
+  }
```

Comment:
Why is this CSS needed? It does not look like it was before when using `watt-card` or `mat-card` instead of a `div`

Reply:
It is not applying the shadows and elevation, by default, when inheriting from the WattCardComponent. In order to keep the styling as it was before, I had to manually specify it in the css.

Reply:
Fixed